### DISCUSSION TITLE
Lerna does no longer build my app module?

### DISCUSSION THREAD

**Discussion by rsoika** [eclipse-glsp/glsp#961] [Q&A] [answered]
I run into a strange situation with my project setup which is aligned to the workflow-example. 
Suddenly my build process does no longer include my -app module 

I need to run `yarn install` manually in the app module each time before I can run `yarn install` on my root directory. 

What could be the reason that lerna is missing my app module? I see the following message:

```
$ yarn install
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
$ lerna run prepare
lerna notice cli v6.5.1

 >  Lerna (powered by Nx)   Running target prepare for 4 projects:

    - @open-bpmn/open-bpmn-glsp
    - @open-bpmn/open-bpmn-model
    - @open-bpmn/open-bpmn-properties
    - @open-bpmn/open-bpmn-theia
```

But I am pretty sure that until some hours ago also the app module was listed by lerna:

```
    - @open-bpmn/open-bpmn-glsp
    - @open-bpmn/open-bpmn-model
    - @open-bpmn/open-bpmn-properties
    - @open-bpmn/open-bpmn-theia
    - @open-bpmn/open-bpmn-app  # Is missing suddenly
```

I know this sounds idiotic when I say I have not changed anything. But I can't figure out what causes this situation. Has anybody a tip for me?

**Comment by tortmayr** [comment, marked as answer]
I guess you indirectly updated the lerna/nx versions of your yarn.lock file.
Newer version of lerna/nx apparently consile the `.gitignore` file and don`t build packages whose root folder is ignored.
We run into a similar problem recently: https://github.com/eclipse-glsp/glsp/issues/952#issuecomment-1463472294

Luckily the fix for his is rather simple: https://github.com/eclipse-glsp/glsp-theia-integration/commit/00910e5751cd0c8fac57e969d751c108594b27a8

**Comment by rsoika** [comment]
Wow!!! the .gitignore was the problem - thanks!
